README.md

```diff
@@ -1,17 +1,19 @@
 # dogecoin-bootstrap
 
-## 2020-01-29: Latest torrent file for a up to date chain (uncompressed) available at https://dogecoin.gg/dogecoin-bootstrap-2021-01-29.torrent (Hosted/Maintained by https://github.com/IncognitoJam/IncognitoJam )
+## 2021-05-09: Latest torrent file for a up to date chain (uncompressed) available at https://dogecoin.gg/files/dogecoin-bootstrap-2021-05-09.torrent (Hosted/Maintained by https://github.com/IncognitoJam/IncognitoJam)
+
+You can see the wiki to set up a node on a VPS.
 
 Yoinked instructions:
 ```
 You can use the bootstrap to get started more quickly with Dogecoin Core, you don't have to wait as long to synchronise the entire chain.
 
-Make sure you are using the latest Dogecoin Core 1.14.2! Run the wallet once and then close it.
+Make sure you are using the latest Dogecoin Core 1.14.3! Run the wallet once and then close it.
 
-Download: https://dogecoin.gg/dogecoin-bootstrap-2021-01-29.torrent
-Torrent Hash: 9aaaa5c4bd18686d49d6fce7758409349c62567b
+Download: https://dogecoin.gg/files/dogecoin-bootstrap-2021-05-09.torrent
+Torrent Hash: 189a56fdda6be8608df3c62b1915b0181acf846e
 
-After downloading the .torrent, open it with your torrent client of choice, and download it. It's around 45GB.
+After downloading the .torrent, open it with your torrent client of choice, and download it. It's around 40GB.
```

Comment:
Did the size reduce? Anyway, rather give too big of an estimate. Please undo change.

Reply:
Yes. They changed method and are nw providing bootstrap.dat file. I'm not sure what are the advantages / drawbacks about it.